src/ec/packages/ec-component-site-header-harmonised/ecl-site-header-harmonised.html.twig

```diff
@@ -293,7 +293,7 @@
   </div>
 {% endif %}
 {% if _menu is not empty %}
-  {% include '@ecl-twig/ec-component-menu-harmonised/ecl-menu-harmonised.html.twig' with _menu|merge({
```

Comment:
Maybe you noticed this already, looks like a "regression"..
![image](https://user-images.githubusercontent.com/14326885/71085141-1be51e00-2197-11ea-9729-197f3a588080.png)

Reply:
Solved with following merged PR:
https://github.com/ec-europa/ecl-twig/pull/272